src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/RealWorldContextTests.cs

```diff
@@ -303,6 +303,61 @@ public virtual void RoundTripWithCustomPropertyConverterFactory_Struct()
             }
         }
 
+        [Fact]
+        public virtual void RoundTripConcreteDerivedClass()
```

Comment:
I recommend sharing this test with the dynamic serializer in this file instead - https://github.com/dotnet/runtime/blob/7a58bef7497f70ab6bb98efc5f0b3f6b4ea2f4f2/src/libraries/System.Text.Json/tests/Common/PropertyVisibilityTests.cs. There are no `abstract` property tests there right now.

Reply:
Sure thanks.